app/src/services/object.js

```diff
@@ -235,7 +234,7 @@ const service = {
         public: data.public,
         active: data.active,
         updatedBy: data.userId ?? SYSTEM_USER,
-        lastSyncedDate: new Date()
+        lastSyncedDate: data.lastSyncedDate ? data.lastSyncedDate : undefined
```

Comment:
This can be simplified to the following (assuming this doesn't accidentally nullify any values that are already there previously:
```js
lastSyncedDate: data.lastSyncedDate ?? undefined
```

Reply:
confirmed that updating with x: undefined is skipped